Week2/src/clients/budgetCheck.js

```diff
@@ -0,0 +1,31 @@
+import Budget from "../models/budget.js";
+
+import lockedSync from "locked-sync"
+const sync = lockedSync();
+
+
+export default async () => {
+
+    const end = await sync();
+    let budgetAmount = 0
+    try {
+        const userBudget = await Budget.find();
+        if (userBudget) {
+            budgetAmount = userBudget[0].amount
```

Comment:
This may crash in this circunstance
```
const userBudget = [];
 userBudget[0].amount
```

Reply:
also, were are you using budgetAmount in your code ? 
I don't see that you are using that variable